### DISCUSSION TITLE
Metadata ranking criteria is vague

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29590] [Q&A] [answered]
I'm making this post because of [this](https://osu.ppy.sh/beatmapsets/2182114/discussion/-/generalAll/total#/4548503) post on a map, where its debated about a certain rule of the Metadata RC.
There shouldn't be a situation where people have to guess the meaning of the ranking criteria, and it should be clearly stated what rules are optional and what aren't.

I'll look into the RC more when I have time, as I'm writing this in a rush and posted this so it can be acknowledged.

**Comment by Joehuu** [comment, marked as answer]
This is the wrong place to talk about the ranking criteria. Maybe your issue will be fine in https://osu.ppy.sh/community/forums/topics/830264 or even https://github.com/ppy/osu-wiki/issues.
